.esdata/config/scripts/contributors_size.groovy

```diff
@@ -0,0 +1 @@
+if (_source.containsKey('github')) _source['github'].contributors.size() else 0
```

Comment:
you should not precompile the script. It complexifies the deployment. This would work only with a local instance of Elasticsearch.